pkg/relayer/session/sessiontree.go

```diff
@@ -21,11 +22,11 @@ type sessionTree struct {
 	// sessionMu is a mutex used to protect sessionTree operations from concurrent access.
 	sessionMu *sync.Mutex
 
-	// sessionHeader is the header of the session corresponding to the SMST (Sparse Merkle State Tree).
+	// sessionHeader is the header of the session corresponding to the SMST (Sparse Merkle State Trie).
 	sessionHeader *sessiontypes.SessionHeader
 
-	// tree is the SMST (Sparse Merkle State Tree) corresponding the session.
-	tree *smt.SMST
+	// trie is the SMST (Sparse Merkle State Trie) corresponding the session.
+	trie smt.SparseMerkleSumTrie
```

Comment:
Thoughts on called this `smt` instead?

1. This file is called `sessiontree`
2. We have a `treestore`
3. I can see people getting a bit confused of "when to use tree vs trie`
4. `smt` captures both in one and avoid cognitive overload and/or ambiguity.